client/stylesheets/_toCat.scss

```diff
@@ -3123,6 +3123,10 @@ On click the images will be displayed at normal size to complete the effect
     font-size: 1.1rem; 
 }
 
+.listingItemDetailItems li a:hover {
+    color: #5d5d5d;
```

Comment:
Color literals like `#5d5d5d` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces